warehouse/query-core/src/main/java/datawave/query/iterator/pipeline/PipelineIterator.java

```diff
@@ -298,6 +298,7 @@ public void logYieldKeyHistory(WaitWindowOverrunException e, String prefix) {
         for (Pair<Key,String> h : e.getYieldKeyHistory()) {
             log.debug(prefix + " yieldKeyHistory-" + n++ + ": " + h);
         }
+        log.trace("Yielded", e);
```

Comment:
```suggestion
        log.trace(prefix, e);
```

Reply:
Called both when yielding immediately and when returning a WaitWindowOverrun marker (if collectTimingDetails is true).